### ISSUE TITLE
No such file or directory

### ISSUE THREAD

**Issue by santhosh-umapathi-appymango** [ionutvmi/uglifyjs-folder#34]
I was trying to minify the folder dist/src on the root level of the project and tried to get the output in the same folder. But no matter what path I change, I always end up with this error.

```
env: node\r: No such file or directory
error Command failed with exit code 127.
```

Script which I tried,
`uglifyjs-folder dist/src -o dist/src/index.js`

Version Details:
`"uglifyjs-folder": "3.1.3"
`

**Comment by PhilBroadbent** [comment]
I also have this issue - I believe it's caused by the files having windows style line endings, so the owner will have to change that

This actually broke from v3.0.0 to v3.1.0 (and 3.1.0 broke commonjs compatibility so verified on 3.1.2)

FYI running `dos2unix` on `cli.js` fixes it (temporarily), but until it's fixed, best to downgrade to v3.0.0

I did try convert files but Git doesn't see difference so I assume it happens during publish

I found this issue here https://github.com/npm/npm/issues/2097, and through that (near bottom) another package having issues, and they fixed it  by this change: https://github.com/pattern-lab/patternlab-node/pull/1397/files#diff-6e2e2a1851648938b325ba84de634407a4e69a644ea61102df15ca4a8a7a9758R51 so owner may need to do this, too.

**Comment by ionutvmi** [comment]
I published version 3.1.4 to npm which has the correct line endings for the cli.js file.

Please try it out and let me know if you still face issues.

For future reference can you also post some details about the environment where this issue was reproducing (os, node version) ?

**Comment by PhilBroadbent** [comment]
@ionutvmi Looks good, thanks for quick fix
